driver-mct/cime_config/config_component_e3sm.xml

```diff
@@ -408,6 +408,8 @@
       <!-- a shorter timestep is needed for stability in MMF  -->
       <!-- Use for all "low res" grids (i.e. ne4, ne30, ne45) -->
       <value compset="_EAM%MMF">72</value>
+      <!-- Shorter timestep needed for SCREAMv1 because we are not substepping SHOC and P3 yet -->
+      <value compset="_SCREAM.*">288</value>
```

Comment:
So this is using a 5 min dt for all SCREAM compsets regardless of model grid? And previously we defaulted to a half hour dt if we didn't explicitly choose dt? Yeah, this sounds like a good change until we get substepping working.

Reply:
Right. Well, almost. The previous behavior would key off the entries above that set based on resolution, since no defaults were set for SCREAM compsets. So actually for ne4 it would default to a 2-hour timestep based on this entry:
```
      <value compset=".+" grid="a%ne4np4">12</value>
```
we were just overriding that with the `scream-30min_ts` testmod to get 30 minutes. For ne30, we would have gotten 30 minutes because the overall default if no xml keys are matched was 48.

Reply:
👍